src/main/java/seedu/internbuddy/logic/commands/UpdateCommand.java

```diff
@@ -0,0 +1,105 @@
+package seedu.internbuddy.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.internbuddy.logic.parser.CliSyntax.PREFIX_APP_STATUS;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import seedu.internbuddy.commons.core.index.Index;
+import seedu.internbuddy.commons.util.ToStringBuilder;
+import seedu.internbuddy.logic.Messages;
+import seedu.internbuddy.logic.commands.exceptions.CommandException;
+import seedu.internbuddy.model.Model;
+import seedu.internbuddy.model.application.AppStatus;
+import seedu.internbuddy.model.application.Application;
+import seedu.internbuddy.model.company.Company;
+
+/**
+ * Update the status of an application identified using it's displayed index from the address book.
+ */
+public class UpdateCommand extends Command {
+
+    public static final String COMMAND_WORD = "update";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD
+            + ": Updates the status of an application. "
+            + "Parameters: COMPANY_INDEX (must be a positive integer) "
+            + "APPLICATION_INDEX (must be a positive integer) "
+            + PREFIX_APP_STATUS + "APP_STATUS\n"
+            + "Example: " + COMMAND_WORD + " 1 1 "
```

Comment:
Not sure if we want to make the indexes to have prefixes like in the `WithdrawCommand` in #108 to be consistent.